### PR TITLE
added: wishlist functionality

### PR DESCRIPTION
**Pull request by rkapoor10**
added wishlist functionality

features
- add into/remove from wishlist from the product page
- add into/remove from wishlist from cart page
- wishlist page to add or remove items by clicking on the heart icon

### CODE REVIEW

**Review by paarthbajaj — APPROVED** [review]
Logic looks good. The code looks clean. You can merge.

File: src/components/wishlist/WishlistProducts.jsx [hunk 1]
```diff
@@ -1,119 +1,71 @@
...
-                <BsHeartFill className="basered"/>
+              <div className="title">
+                <h3>{title}</h3>
+                <p className="title-small txt-s">
+                  {brand} <span className="gray">{categoryName}</span>
+                </p>
+              </div>
+              <p className="price-box txt-xs fw-bold">
+                Rs. {DiscountedPrice(price, discount)}
+                <s className="striked">Rs. {price}</s>
```
Comment (JantuDeb) [inline src/components/wishlist/WishlistProducts.jsx:52]: convert price to locale string

File: src/components/filters/Filter.jsx [hunk 2]
```diff
@@ -62,8 +62,9 @@ const FilterSidebar = () => {
                     id="checkbox-1"
```
Comment (JantuDeb) [inline src/components/filters/Filter.jsx:62]: you can use `categoryName` for id since it is unique for each checkbox

**Review by JantuDeb — APPROVED** [review]
This is really good. I left few comments do check it.

File: src/components/wishlist/WishlistProducts.jsx (hunk 1 above)
Comment (rkapoor10) [reply src/components/wishlist/WishlistProducts.jsx:52]: will add

File: src/components/filters/Filter.jsx (hunk 2 above)
Comment (rkapoor10) [reply src/components/filters/Filter.jsx:62]: done

**Pull request merged**